install.lua

```diff
@@ -80,10 +83,11 @@ if is_windows then
 "%s" -e "package.path=[[%%~dp0..\src\?.lua;%%~dp0..\src\?\init.lua;]]..package.path" "%%~dp0luacheck.lua" %%*
 ]=]):format(args.lua))
 else
-   fh:write(([=[
+    fh:write(([=[
 #!/bin/sh
+cd "%s"
```

Comment:
So, file paths will be treated relative to the args.source directory?

Reply:
It was a mistake. I removed the line. Now it is work just fine due to package.path, where paths to luacheck sources is stored. These paths are added to package.path during package building, also it is original luacheck way, so I think it's ok.